### ISSUE TITLE
EditableDialogBoxConfiguration: impossible to clear relation

### ISSUE THREAD

**Issue by NiklasBr** [pimcore/pimcore#7485]
# Bug Report

### Expected behavior

After adding a relation in the EditableDialogBoxConfiguration popup window, it should be possible to clear it.

### Actual behavior

There is no button to click to empty the content, and backspace/delete does not work in the field.

### Code example
```php
<?php
declare(strict_types=1);

namespace AppBundle\Document\Areabrick;

use Pimcore\Extension\Document\Areabrick\EditableDialogBoxConfiguration;
use Pimcore\Extension\Document\Areabrick\EditableDialogBoxInterface;
use Pimcore\Model\Document;
use Pimcore\Model\Document\Editable\Area\Info;

class Mapbrick implements EditableDialogBoxInterface
{
    protected int $locationCount = 10;

    public function getEditableDialogBoxConfiguration(Document\Editable $area, ?Info $info): EditableDialogBoxConfiguration
    {
        $config = new EditableDialogBoxConfiguration();
        $config->setHeight(700);
        $config->setWidth(600);
        $config->setReloadOnClose(true);

        for ($i = 0; $i++ < $this->locationCount;) {
            $config->addItem([
                'type' => 'relation',
                'label' => 'brick.label.geo-location' . ' ' . $i,
                'name' => 'geopoint_relation_' . $i,
                'config' => [
                    'width' => 300,
                ],
            ]);
        }

        return $config;
    }
}

```

**Comment by NiklasBr** [comment]
Just to clarify this issue, button to clear does not exist:

![image](https://user-images.githubusercontent.com/279826/105323636-a1bd6f00-5bca-11eb-86e3-4f4a47303217.png)


compare to the regular DataObject relation editor:
![image](https://user-images.githubusercontent.com/279826/105323675-b00b8b00-5bca-11eb-9cc7-1910088207a4.png)

**Comment by brusch** [comment]
Fixed by #8048
